geomesa-process/src/main/scala/org/locationtech/geomesa/process/Point2PointProcess.scala

```diff
@@ -40,7 +40,10 @@ class Point2PointProcess extends VectorProcess {
                minPoints: Int,
 
                @DescribeParameter(name = "breakOnDay", description = "Break connections on day marks")
-               breakOnDay: Boolean
+               breakOnDay: Boolean,
+
+               @DescribeParameter(name = "filterSingularPoints", description = "Filter on groups that all fall on the same point", defaultValue = "true")
```

Comment:
It's a geomesa bug - when you compose this with Import, geomesa geohashing throws an exception on a line with no distance.